aws/resource_aws_cloudwatch_event_target_test.go

```diff
@@ -730,6 +765,89 @@ resource "aws_sns_topic" "test" {
 `, ruleName, snsTopicName)
 }
 
+func testAccAWSCloudWatchEventTargetConfig_retrypolicy_dlc(ruleName, targetName, rName string) string {
```

Comment:
This should use camal case for everything past the `_`. eg `testAccAWSCloudWatchEventTargetConfig_retryPolicyDlc`